algorithme-evaluation.md

```diff
@@ -90,9 +93,70 @@ Le modèle est entrainé sur les variables d'apprentissage suivantes:
 
 Voir [ce document](https://github.com/signaux-faibles/opensignauxfaibles/blob/master/js/reduce.algo2/docs/variables.json) pour la définition et la source des champs présents en base.
 
-### Seuils de détection :construction_worker:
+### _Evaluation du modèle: lexique_
 
-Les seuils ont pour l'instant été arbitrairement fixés pour garantir des volumes raisonnables d'entreprises détéctées. Il serait pertinent d'améliorer cette méthode, notamment en trouvant les seuils qui maximisent notre critère d'évaluation.
+Pour notre problème, nous choisissons d'attribuer un score de 1 aux entreprises ayant un risque maximal de défaillance, et 0 aux entreprises ayant un risque minimal de défaillance. En conséquence, nous avons les définitions suivantes:
+
+Défaillance
+: une entrée en procédure collective, ou une dette urssaf maintenue plus de 3 mois
+
+Elément/établissement positif
+: un établissement connaissant effectivement une défaillance à au moins un moment sur les 18 prochains mois
+
+Elément/établissement négatif
+: le contraire d'un établissement positif
+
+Elément/établissement prédit positif
+: un établissement que notre algorithme identifie comme à risque de défaillance (fort ou modéré) sur les 18 prochains mois
+
+Elément/établissement prédit négatif
+: le contraire d'un établissement prédit positif
+
+Elément/établissement faux positif
+: un établissement pour lequel une défaillance est prédite, mais qui ne connaît pas de défaillance effective
+
+Elément/établissement faux négatif
+: un établissement pour lequel aucune défaillance n'est prédite, mais qui connaît effectivement une défaillance
+
+A partir de ces définitions, on dérive les métriques usuelles d'évaluation d'un algorithme d'apprentissage automatique:
+
+Précision
+: la part d'établissements prédits positifs étant effectivement positifs
+
+Rappel
+: la part d'établissements effectivement positifs étant prédits positifs
+
+Score $F_{\beta}$
+: une métrique d'évaluation prenant à la fois la précision et le rappel en compte, et accordant une importante relative $\beta$ fois plus importante au rappel qu'à la précision
+
+Seuil de classification
+: le modèle Signaux Faibles produit un score de risque entre 0 et 1 produit par notre algorithme. Hors, il faut décider à quelle pallier de risque appartient chaque établissement à partir de ce score de risque. Pour ce faire, et au vu du modèle et de la cible d'apprentissage actuelle, il est nécessaire de définir un premier seuil sur le score de risque au-delà duquel l'établissement est à considérer "à risque modéré" de défaillance, et un second seuil, plus élevé que le premier, au-delà duquel l'établissement est à considérer "à risque fort" de défaillance. La méthodologie pour déterminer ces seuils est détaillée ci-dessous
```

Comment:
```suggestion
: le modèle Signaux Faibles produit un score de risque entre 0 et 1 produit par notre algorithme. Hors, il faut décider à quel pallier de risque appartient chaque établissement à partir de ce score de risque. Pour ce faire, et au vu du modèle et de la cible d'apprentissage actuelle, il est nécessaire de définir un premier seuil sur le score de risque au-delà duquel l'établissement est à considérer "à risque modéré" de défaillance, et un second seuil, plus élevé que le premier, au-delà duquel l'établissement est à considérer "à risque fort" de défaillance. La méthodologie pour déterminer ces seuils est détaillée ci-dessous
```

Reply:
Oups, some spelling mistakes!
Fixed in https://github.com/signaux-faibles/documentation/pull/45/commits/aca623779b339ee2d603892b966e652b5a56d659